components/builder-admin/habitat/plan.sh

```diff
@@ -1,16 +1,25 @@
 pkg_name=hab-builder-admin
 pkg_origin=core
-pkg_version=$(cat "$PLAN_CONTEXT/../../../VERSION-BLDR")
+pkg_version=undefined
 pkg_maintainer="Jamie Winsor <reset@chef.io>"
 pkg_license=('Apache-2.0')
 pkg_source=nosuchfile.tar.gz
 pkg_bin_dirs=(bin)
 pkg_deps=(core/glibc core/openssl core/gcc-libs core/zeromq core/libsodium core/libarchive)
 pkg_build_deps=(core/protobuf core/protobuf-rust core/coreutils core/cacerts
-  core/rust core/gcc core/pkg-config)
+  core/rust core/gcc core/git core/pkg-config)
 bin="bldr-admin"
 pkg_svc_run="$bin start -c ${pkg_svc_path}/config.toml"
 
+do_verify() {
```

Comment:
I chose `do_verify()` here to populate the version because this happens before `_build_environment` which is the first place that we *need* a proper pkg_version set.

/cc @fnichol do you think it makes sense to have a callback with a more proper name which is for determining a version number lazily? Maybe that's just overkill.